src/Infrastructure/Identity/RoleService.cs

```diff
@@ -165,7 +169,7 @@ public async Task<string> RegisterRoleAsync(RoleRequest request)
     /// <param name="selectedPermissions"></param>
     /// <param name="cancellationToken"></param>
     /// <returns></returns>
-    public async Task<string> UpdatePermissionsByRoleAsync(string roleId, List<UpdatePermissionsRequest> selectedPermissions, CancellationToken cancellationToken)
+    public async Task<string> UpdatePermissionsAsync(string roleId, List<UpdatePermissionsRequest> selectedPermissions, CancellationToken cancellationToken)
     {
         var errors = new List<string>();
```

Comment:
this one can then be removed

Reply:
Why? this one is the currently used to update

Reply:
I meant the errors list... can immediately parse the list into the exception... no need to addrange first...